### PR TITLE
Remove show routes on map

### PR DESCRIPTION
**Pull request by landonreed**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Removes the editor map layer control to show routes on the map. Fixes ibi-group/datatools-ui#627. Commented out because there is client interest in this feature coming back, but at the moment it's too risky.

### CODE REVIEW

**Review by evansiroky — APPROVED** [review]
I'll approve this for now, but once this is merged and it closes #627, the issue should be reopened because this merely suppresses the problem instead of fixing the underlying problem. It'd probably be a good idea to encode polylines on the backend so that various calls to retrieve shape data won't require outputting raw shape data.

**Review by binh-dam-ibigroup — APPROVED** [review]

**Pull request merged**
